src/session.rs

```diff
@@ -194,18 +194,19 @@ impl Session {
         &self.app
     }
 
-    pub fn set_keying_material(&mut self, mat: KeyingMaterial, active: bool) {
+    pub fn set_keying_material(
+        &mut self,
+        mat: KeyingMaterial,
+        srtp_profile: SrtpProfile,
+        active: bool,
+    ) {
+        // TODO: rename this to `initialise_srtp_context`?
```

Comment:
I guess based on your comment about the log message you don't agree with this @algesten?

Reply:
Bit of a sideways change. Don't quite see the reason.

Reply:
I have thoughts. Let's leave it for now.

Reply:
Cool, I'll drop the comment